src/services/metrics-recorder.ts

```diff
@@ -116,14 +116,22 @@ export class MetricsRecorder {
         fallbackTag = ' FALLBACK'
       }
 
+      // Save raw methods for the errors db
+      const rawMethods = method
+
+      // Reduce multi-method calls for metrics/logging purposes
+      if (method && method.split(',').length > 1) {
```

Comment:
> 

It was up there because the logs were supposed to be using this simplified method too. It was later decided to not too for now, so your proposed changes make sense.

Addressed in f6f49a252dc5d5b61eb1446673ddaefb226cce7d